docs/spec.md

```diff
@@ -1415,9 +1421,9 @@ where:
 
 #### Outputs
 
-| Name      | Type                                 | Constraints |
-|-----------|--------------------------------------|-------------|
-| `results` | variadic number of tensors or tokens | (C4)        |
+| Name      | Type                                                               | Constraints |
+|-----------|--------------------------------------------------------------------|-------------|
+| `results` | variadic number of tensors, per-tensor quantized tensors or tokens | (C4)        |
```

Comment:
Any particular reason to not allow per-axis quantized tensors for control flow ops? 

Similarly to e.g. tokens, if there are StableHLO ops which take per-axis quantized tensors, then I think control flow ops should support them. Otherwise, we'd be able to express `op(foo)` but not `bar = cond ? foo1 : foo2; op(bar)` which I think would be unfortunate.

I think that the same logic applies to ConstantOp/InfeedOp/RecvOp which get values from the outside world and OutfeedOp/SendOp which put values into the outside world. If we allow per-axis quantized tensors in parameters and returns, I think we should allow them in the ops mentioned above.